include/Parser/Factory.hpp

```diff
@@ -17,6 +17,7 @@
 
 #include "Parser/DLLoader.hpp"
 #include "Primitives/Sphere.hpp"
+#include "Primitives/Plane.hpp"
```

Comment:
Why include Plane.hpp in this file ?

Reply:
Like I include sphere it's for dynamic cast